model/installation.go

```diff
@@ -50,6 +50,7 @@ type InstallationFilter struct {
 	Page           int
 	PerPage        int
 	IncludeDeleted bool
+	Name           string
```

Comment:
Should we use `DNS` over `Name` to be more clear? Because this will be `somename.cloud.mattermost.com` and not just `somename`, right?

Reply:
Yeah that's fine with me. You're correct that it's the full FQDN.